news.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Media Strategy
+tagline: Project Jupyter is committed to ensuring communication in Jupyter's official channels is strategic and benefit Project Jupyter.
+permalink: /news
+---
+
+The Jupyter Media Strategy Working Group (JMS) exists to ensure that communications in Jupyter official channels are strategic and benefit Project Jupyter.
+
+## Jupyter Blog
+The Jupyter blog (blog.jupyter.org) is designed for community members to share valuable, educational content, including news and tutorials, about Jupyter.  Please see the Jupyter Blog Submission [Guidelines](./blog_guidelines) for details on creating a post.
+
+## Jupyter Discourse Forum 
+The Discourse forum (discourse.jupyter.org) is meant to be a welcoming place for discussion about Project Jupyter.  Members are encouraged to ask and answer questions, and highlight wins. If you are interested to be a part of the Jupyter Discourse moderation team, please reach out to us at jupyter-media-strategy at googlegroups dot com.
+
+## Social Media Accounts
+The JMS maintains a list of official Project Jupyter social media accounts to support and promote our organization's interests.  We primarily use our Mastodon account for interactive social media engagement, but will also broadcast messages about the blog or Jupyter events on other channels. 
+
+If you are organizing a Jupyter event or have something Jupyter-related that should be boosted by the official Jupyter account, please reach out to us at jupyter-media-strategy at @googlegroups dot com.
+
+List of channels maintained by JMS in alphabetical order:
+- Facebook: facebook.com/projectjupyter
+- LinkedIn: linkedin.com/company/project-jupyter
+- Mastodon:  https://hachyderm.io/@ProjectJupyter
+- Slack JupyterCon 2023: jupyterconworkspace.slack.com
+- Slack JupyterLab: jupyterlabworkspace.slack.com
+- X/Twitter Jupyter: twitter.com/ProjectJupyter
+- X/Twitter JupyterCon: twitter.com/JupyterCon
+- YouTube: youtube.com/@ipython
```

Comment:
Most of these are missing https://

We shouldn't be promoting the Slack JupyterCon 2023 or JupyterCon accounts until we have an upcoming conference to promote. If you want these to be in here for historical reasons, they should be in a section of their own.

Reply:
Fixed